packages/server/engine-lib/rocketlib-python/lib/rocketlib/engine.py

```
@@ -107,6 +107,23 @@ def monitorOther(*args) -> None:
 globals()['monitorOther'] = engLib.monitorOther  # noqa
 
 
+def monitorSSE(pipe_id: int, type: str, message: str, data: dict = None) -> None:
```

Ruff (0.15.5)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)